ffmpeg/api_test.go

```diff
@@ -877,3 +877,79 @@ func consecutiveMuxerOpts(t *testing.T, accel Acceleration) {
 	run(cmd)
 
 }
+
+func TestTranscoder_EncodingProfiles(t *testing.T) {
+	encodingProfiles(t, Software)
+}
+
+func encodingProfiles(t *testing.T, accel Acceleration) {
```

Comment:
Curious, should we add an interlaced test case for constrained high? Even though we know the behavior is incorrect, it might help to codify that until we get a more permanent solution in. Something along the lines of this https://github.com/livepeer/lpms/blob/ec8e1e4e242024eaa0a7c83cfe67fe20a4d34d11/ffmpeg/ffmpeg_test.go#L1104-L1110

Reply:
Actually I think we can safely say the behaviour as correct. Testing further with `ffprobe -show_frames` showed me that the `interlaced_frame` flag is unset by default on all profiles, unless we [explicitly enable](https://stackoverflow.com/questions/22471099/how-to-convert-a-1080p-to-1080i-using-ffmpeg) MBAFF during encoding via `-flags +ilme+ildct -x264opts tff=1`. (The video still looks stripey if original content was interlaced regardless of if MBAFF is used or not. The only way to prevent that is still an expensive deinterlacing filter before encoding)

I will add a testcase to assert that the `interlaced_frame` parameter is unset for baseline & constrained high.

Reply:
I'm just waiting for the day until someone sends in interlaced content and complains about stripey output. Until then, we can say it's working as intended...